setup.py

```diff
@@ -128,7 +128,7 @@ def index():
         'honeybadger==0.1.2',
```

Comment:
Let's loosen the minor/path versions of `honeybadger` dependency too and test it in POrch's web process using a `1.0.0.alpha` version before releasing `1.0.0`